src/UIState/MainMenu.cpp

```diff
@@ -203,7 +199,7 @@ void MainMenu::selectSet() {
       this->setNextState((UIState *)new SetKP(tc));
       break;
     case SET_PID_ON_OFF:
-      this->setNextState((UIState *)new PIDTuningMenu(tc));
+      this->setNextState((UIState *)new SetPIDOnOff(tc));
```

Comment:
We have a new code style checker that is stricter about some things that we have been in the past. This line should be revised as follows:
`   this->setNextState(static_cast<UIState *>(new SetPIDOnOff(tc)));`
Eventually all the other "C-style type casts" should be converted to "C++-style type casts" but for now I'm just doing them one-at-a-time as the tests complain.